recipes/fiftyone/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "fiftyone" %}
+{% set version = "0.24.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiftyone-{{ version }}.tar.gz
+  sha256: 451b4f24496ee610bd76d0d344722a4e71c7636c71d80e9e11813bf330bfe44e
+  patches:
+    - 0000-relax-deps.patch
+    - 0000-dropin-fiftyone-db.patch
+    - 0000-allow-newer-strawberry.patch
+
+build:
+  skip: true  # [py<37 and py>312]
```

Comment:
You can make this one noarch by releasing the importlib-metadata  pin below. It is fine to install it in py>=38.

Reply:
Done

Reply:
It is not noarch. You need to add:

`noarch: python` here. Please see https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-builds